ci_tests/src/accountant.ts

```diff
@@ -0,0 +1,43 @@
+import {
+  getWallet,
+  getWormchainSigningClient,
+} from "@wormhole-foundation/wormchain-sdk";
+import { ZERO_FEE } from "@wormhole-foundation/wormchain-sdk/lib/core/consts";
+import { toUtf8 } from "cosmwasm";
+
+// cache the client and signer
+let client: Awaited<ReturnType<typeof getWormchainSigningClient>>;
+let signer: string;
+
+export async function submitAccountantVAA(vaa: Uint8Array) {
+  if (!signer) {
+    // NttAccountantTest = wormhole18s5lynnmx37hq4wlrw9gdn68sg2uxp5rwf5k3u
+    const wallet = await getWallet(
+      "quality vacuum heart guard buzz spike sight swarm shove special gym robust assume sudden deposit grid alcohol choice devote leader tilt noodle tide penalty"
```

Comment:
I'm going to assume this seed phrase is fine being out in the wild?

Reply:
Yes it’s a default cosmos one and in the WH repo